gravitee-am-test/specs/gateway/mfa.jest.spec.ts

```diff
@@ -271,8 +302,6 @@ describe("MFA", () => {
             expect(authorize2.headers['location']).toContain(`${process.env.AM_GATEWAY_URL}/${domain.hrid}/mfa/challenge`);
 
             const successfulVerification = await verifyFactor(authorize2, validMFACode, smsFactor);
-            expect(successfulVerification.headers['location']).toBeDefined();
-            expect(successfulVerification.headers['location']).toContain(`${process.env.AM_GATEWAY_URL}/${domain.hrid}/oauth/authorize`);
```

Comment:
Instead of removing these two checks, why not doing them into the verifyFactor method (at least the last one to test the right Location value) ?

Reply:
See line 671.

Reply:
ok, my bad. :+1: